Ejercicio_Palindromo.py

```diff
@@ -0,0 +1,30 @@
+# Encontrar si una cadena de texto es palindromo.
+# Consideraciones: puede ser palabra, oracion.
+# Crear funcion, solo utiliza 1 cadena de texto
+
+
+def palindromo(texto):
+
+    string1 = ''
+    string2 = ''
+
+    texto = texto.lower()
+
+    for i in range(len(texto)):
+        if texto[i] == ' ':
+            pass
+        else:
+            string1 += texto[i]
```

Comment:
Estas 5 líneas del FOR se pueden simplificar a 3

Reply:
Si quieres un desafío más grande, toda la función se puede simplificar a unas 4 lineas aprox